setup.py

```diff
@@ -25,7 +25,7 @@ def get_version() -> str:
     long_description_content_type="text/x-rst",
     author="Gauvain Pocentek",
     author_email="gauvain@pocentek.net",
-    license="LGPLv3",
+    license="LGPL-3.0-only",
```

Comment:
@max-wittig WDYT before I merge this, we don't have a `This project is licensed under the terms of the <something> license.` so from what I can tell it's not even clear (even from Gauvain's initial comments and text), and we should decide right here on `LGPL-3.0-only` vs `LGPL-3.0-or-later`.

The proposed PEP even doesn't want to disambiguate this automatically (https://peps.python.org/pep-0639/#mapping-license-classifiers-to-spdx-identifiers).

Reply:
I guess or later is fine?

Reply:
Works for me as well.

```suggestion
    license="LGPL-3.0-or-later",
```

Reply:
@RazerM are you still interested and would like to apply the suggested change above?

Reply:
Done

Reply:
Thanks for the PR and the discussion above!